### ISSUE TITLE
Problème date de visionnage

### ISSUE THREAD

**Issue by Tilomb** [phileastv/SensBoxd#3]
Merci pour ce super outil mais il y a un problème dans les dates de visionnages. Elles sont un jour en retard. Un film que j'ai vu le 5 décembre apparait le 4 décembre dans le fichier.

**Comment by phileastv** [comment]
Salut @Tilomb,

Ah mince, c'est vraiment possible. Il faut que je regarde ! En attendant, il y a probablement moyen [que tu règles ça pour ton cas](https://www.) avec des formules excel !

Je regarde pour régler ça pour tout le monde prochainement.

**Comment by Tilomb** [comment]
Oui j’ai réglé le problème moi même sur Excel

**Comment by fizzmizer** [comment]
Même problème pour moi, toutes les dates sont décalées d'un jour. Je pourrais aussi retravailler le .csv pour corriger, mais j'ai aussi dû encoder manuellement une cinquantaine de films qu'il ne trouvait pas donc j'hésite. Il a du mal avec les transcriptions coréennes/japonaises de certains films.

**Comment by phileastv** [comment]
Merci pour ces retours @fizzmizer , si tu peux me donner ton id SensCritique & les films sur lesquels ça bloque, je pourrai essayer de faire une nouvelle version avec ces corrections prochainement.

**Comment by fizzmizer** [comment]
Hello et merci. Mon id SensCritique, c'est fizzmizer.

En gros, j'ai remarqué que ça bloque pour les films coréens et japonais identifiés sur SC par une transcription en caractères latins. J'ai tout encodé manuellement donc je n'ai plus de liste complète, mais voici un exemple parmi d'autres : 
https://www.senscritique.com/film/jsa_joint_security_area/404331

Sur letterbox, son titre original n'est pas transcrit : https://letterboxd.com/film/joint-security-area/

**Comment by phileastv** [comment]
Ce que tu veux dire du coup c'est que le film se retrouve bien dans le CSV avec le nom Français, mais que letterbox ne l'importe pas correctement parce qu'il ne comprend pas le nom en caractere latin c'est ça ?

Si c'est ça, ça risque d'être un peu compliqué à régler malheureusement parce que je vois que sens critique a un champ de nom original, mais uniquement en caractère latin. Donc a moins de récupérer un id que letterboxd comprend mieux (par IMDb/Tmdb par exemple), ça risque d'être compliqué _- mais potentiellement ça sera plus fiable donc je vais y jeter un oeil._

**Comment by fizzmizer** [comment]
Oui, en gros c'est bien ça : le film se retrouve dans le csv avec le titre en version originale transcrite alors qu'ils sont indiqués en caractères japonais/coréens sur letterbox. Effectivement, ça me semble compliqué à automatiser, mais en ce qui me concerne, j'ai réglé ça à la main.

Encore merci pour le taf en tout cas !

**Comment by CarinaChenot** [comment]
Salut @phileastv j’ai fait une PR pour fixer le problème de dates https://github.com/phileastv/SensBoxd/pull/7 😊

**Comment by phileastv** [comment]
Hello ! Merci à vous pour le partage de tous ces détails, et à @CarinaChenot pour le fix en PR 🫶
